src/Components/IndexScene/LayoutScene.tsx

```diff
@@ -1,26 +1,55 @@
 import { GrafanaTheme2 } from '@grafana/data';
-import { SceneComponentProps, SceneObjectBase, VariableValueSelectors } from '@grafana/scenes';
+import { SceneComponentProps, SceneObjectBase, SceneObjectState, VariableValueSelectors } from '@grafana/scenes';
 import { useStyles2 } from '@grafana/ui';
 import React from 'react';
 import { PatternControls } from './PatternControls';
 import { AppliedPattern, IndexSceneState } from './IndexScene';
 import { css } from '@emotion/css';
 import { GiveFeedbackButton } from './GiveFeedbackButton';
+import { InterceptBanner } from './InterceptBanner';
+
+interface LayoutSceneState extends SceneObjectState {
+  interceptDismissed: boolean;
+}
+
+const interceptBannerStorageKey = 'interceptBannerStorageKey';
```

Comment:
Can we prefix this with `pluginJson.id`, like we do with other localstorage keys?
https://github.com/grafana/explore-logs/blob/bfd4682c33ecf987679358102cb222922fc92b0f/src/services/store.ts#L3-L4

Reply:
Done